packages/commercetools/composables/src/getters/productGetters.ts

```diff
@@ -112,7 +124,8 @@ const productGetters: ProductGetters<ProductVariant, ProductVariantFilters> = {
   getSku: getProductSku,
   getFormattedPrice,
   getTotalReviews,
-  getAverageRating
+  getAverageRating,
+  getBreadcrumbs: getProductBreadcrumbs
```

Comment:
It should also be added in boilerplate

Reply:
✔️ added [here](https://github.com/vuestorefront/vue-storefront/pull/6111/commits/f266a63fd7801a34d75d010e654517faf631916f)

Reply:
Thanks!